packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -359,9 +383,33 @@ export default class Editor {
             content = extractContentEvent.content;
         }
 
+        if (this.core.inDarkMode && !getInDarkMode) {
+            content = this.getLightModeContent(content);
+        }
+
         return content;
     }
 
+    private getLightModeContent(content: string): string {
+        let el = document.createElement('div');
+        el.innerHTML = content;

```

Comment:
I'm not convinced this is the right way to do this. Is there a way to do like above, but without having to create an element in the dom?